pkg/component/plutono/plutono.go

```diff
@@ -367,31 +453,12 @@ func (p *plutono) getDashboards(ctx context.Context) (map[string]string, error)
 			}
 		}
 
-		// Fetch extensions provider-specific monitoring configuration
-		existingConfigMaps := &corev1.ConfigMapList{}
-		if err := p.client.List(ctx, existingConfigMaps,
-			client.InNamespace(p.namespace),
-			client.MatchingLabels{v1beta1constants.LabelExtensionConfiguration: v1beta1constants.LabelMonitoring}); err != nil {
+		dashboard, err := p.getAdditionalDashboards(ctx, labelSelector, []string{v1beta1constants.PlutonoConfigMapOperatorDashboard, v1beta1constants.PlutonoConfigMapUserDashboard})
```

Comment:
```suggestion
		additionalDashboards, err := p.getAdditionalDashboards(ctx, labelSelector, []string{v1beta1constants.PlutonoConfigMapOperatorDashboard, v1beta1constants.PlutonoConfigMapUserDashboard})
```

Same: Nit: use plural for the name of the variable.